aws/resource_aws_apprunner_custom_domain_association_test.go

```diff
@@ -16,6 +17,11 @@ import (
 )
 
 func TestAccAwsAppRunnerCustomDomainAssociation_basic(t *testing.T) {
+	domain := os.Getenv("APPRUNNER_CUSTOM_DOMAIN")
```

Comment:
i think we had a dictionary for these env vars but I can't remember where to look 😅

Reply:
https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/MAINTAINING.md#environment-variable-dictionary